modules/client-configuration/pages/autoinst-pxeboot.adoc

```diff
@@ -0,0 +1,99 @@
+[[installation-via-pxe-boot]]
+= Installation via PXE Boot
+
+Here is how a network installation goes:
+
+. the client boots in PXE mode;
+. a DHCP server provides the client with an IP address and mask, the address of an installation server, and the name of a bootloader file on that server;
+. the client downloads the bootloader file via the TFTP protocol from the installation server and executes it;
+. the client is presented with a choice of possible distributions to install from a menu, or starts automatically installing one of the distributions;
+. the bootload downloads via the TFTP protocol a kernel and an initial RAM disk for this distribution;
+. the kernel starts an installation program, either {kickstart} or {ay} - from now on it uses resources provided on the server via the HTTP protocol;
+. the distribution is installed automatically according to a {kickstart} or an {ay};
+. the profile calls a code snippet that registers the client to the {productname} Server, either as a traditional client, or as a Salt minion.
+
+image::cobbler_menu.png[scaledwidth=80%]
+
+The installation server can either be the {productname} Server or one of its proxies. To install from a proxy, you must synchronize the TFTP tree between the server and the proxy.
+
+The DHCP server may also provide other configuration information to the client, like a host name, the address of a router and the address of a domain name server. Some of this information might be needed for the autoinstallation, for example if you specify the installation server by its domain name.
+
+In the installation menu, the first of the choices is [guimenu]``Local boot``. Selecting this choice continues the boot process from the local disk drives. This option is selected automatically if no distribution was selected after a certain time. This is a security measure to prevent from starting an autoinstallation if no human operator is there to select one of the distributions.
+
+Alternatively, the installation might start automatically from one of the distributions without manual intervention. This is called "unattended provisioning".
+
+The "bare metal" functionality is a variant of unattended provisioning where the bootloader file only registers the client at the {productname} Server and does not start the installation. You may then trigger a reinstallation as described in previous section xref:client-configuration:autoinst-reinstall.adoc[Reinstalling Registered Systems].
+
+.Procedure: Performing an Installation with PXE Boot
+. Prepare the DHCP server, see xref:client-configuration:autoinst-pxeboot.adoc#prepare-the-dhcp-server[Prepare the DHCP Server].
+. Prepare an autoinstallation distribution, see xref:client-configuration:autoinst-distributions.adoc[Autoinstallation Distributions].
+. Prepare an autoinstallation profile, see xref:client-configuration:autoinst-profiles.adoc[Autoinstallation Profiles].
+. Reboot the client, and chose a distribution to install if needed.
```

Comment:
I don't really like that is presented as a procedure, when it's just a list of links to other procedures (so I guess it's technically a process). Can we rearrange this to make it a little friendlier?

Reply:
One of the problems is this section is pretty long, without much hope to split it.

Having a procedure was a way to have this part condensed in one block.

Maybe Joseph will have a better idea.